pkg/core/runtime/reports/reports.go

```diff
@@ -248,16 +249,24 @@ func (b *reportsBuffer) initImmutable(rt core_runtime.Runtime) {
 
 func startReportTicker(rt core_runtime.Runtime, buffer *reportsBuffer, extraFn core_runtime.ExtraReportsFn) {
 	go func() {
-		err := buffer.dispatch(rt, pingHost, pingPort, "start", extraFn)
+		contexts, err := iterator.CustomIterator()
```

Comment:
Does it make sense to have reports work on the multi-tenant setup. (I don't think so)

Reply:
Ok, will revert.

Reply:
Actually the report function of the KM that counts active dataplanes needs the context to properly count the dps. So should I keep that?

Reply:
it's not 100% sure that we're going to use that in KM so let's revert this